[2016-06-13 03:43:55] local.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Encryption\EncryptionServiceProvider.php:45
Stack trace:
#0 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Encryption\EncryptionServiceProvider.php(25): Illuminate\Encryption\EncryptionServiceProvider->getEncrypterForKeyAndCipher('SomeRandomStrin...', 'AES-256-CBC')
#1 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Container\Container.php(735): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#2 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Container\Container.php(633): Illuminate\Container\Container->build(Object(Closure), Array)
#3 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(697): Illuminate\Container\Container->make('encrypter', Array)
#4 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Container\Container.php(853): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#5 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Container\Container.php(808): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#6 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Container\Container.php(779): Illuminate\Container\Container->getDependencies(Array, Array)
#7 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Container\Container.php(633): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#8 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(697): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#9 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(126): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#10 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#11 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#12 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#13 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Router.php(726): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Router.php(699): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#16 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Router.php(675): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#17 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(246): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#18 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#19 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(52): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#20 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(44): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#21 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#22 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(136): call_user_func_array(Array, Array)
#23 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#24 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#25 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#26 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#27 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(132): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#28 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#29 D:\xampp\htdocs\raovatvnn\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#30 {main}  
[2016-06-13 03:43:55] local.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Encryption\EncryptionServiceProvider.php:45
Stack trace:
#0 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Encryption\EncryptionServiceProvider.php(25): Illuminate\Encryption\EncryptionServiceProvider->getEncrypterForKeyAndCipher('SomeRandomStrin...', 'AES-256-CBC')
#1 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Container\Container.php(735): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#2 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Container\Container.php(633): Illuminate\Container\Container->build(Object(Closure), Array)
#3 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(697): Illuminate\Container\Container->make('encrypter', Array)
#4 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Container\Container.php(853): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#5 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Container\Container.php(808): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#6 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Container\Container.php(779): Illuminate\Container\Container->getDependencies(Array, Array)
#7 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Container\Container.php(633): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#8 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(697): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#9 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(152): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#10 D:\xampp\htdocs\raovatvnn\index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#11 {main}  
[2016-06-16 09:16:32] local.ERROR: exception 'ReflectionException' with message 'Class App\Http\Controllers\Backend\AdminController does not exist' in D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Route.php:280
Stack trace:
#0 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Route.php(280): ReflectionMethod->__construct('App\\Http\\Contro...', 'index')
#1 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Router.php(859): Illuminate\Routing\Route->signatureParameters('Illuminate\\Data...')
#2 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Router.php(844): Illuminate\Routing\Router->substituteImplicitBindings(Object(Illuminate\Routing\Route))
#3 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Router.php(827): Illuminate\Routing\Router->substituteBindings(Object(Illuminate\Routing\Route))
#4 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Router.php(691): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#5 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Router.php(675): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#6 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(246): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#7 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#8 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(52): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#9 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(44): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#11 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(136): call_user_func_array(Array, Array)
#12 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#13 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#15 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#16 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(132): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#17 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#18 D:\xampp\htdocs\raovatvnn\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#19 {main}  
[2016-06-16 09:18:37] local.ERROR: exception 'ReflectionException' with message 'Class App\Http\Controllers\Backend\Backend\IndexController does not exist' in D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Route.php:280
Stack trace:
#0 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Route.php(280): ReflectionMethod->__construct('App\\Http\\Contro...', 'index')
#1 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Router.php(859): Illuminate\Routing\Route->signatureParameters('Illuminate\\Data...')
#2 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Router.php(844): Illuminate\Routing\Router->substituteImplicitBindings(Object(Illuminate\Routing\Route))
#3 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Router.php(827): Illuminate\Routing\Router->substituteBindings(Object(Illuminate\Routing\Route))
#4 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Router.php(691): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#5 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Router.php(675): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#6 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(246): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#7 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#8 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(52): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#9 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(44): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#11 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(136): call_user_func_array(Array, Array)
#12 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#13 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#15 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#16 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(132): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#17 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#18 D:\xampp\htdocs\raovatvnn\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#19 {main}  
[2016-06-16 09:19:51] local.ERROR: exception 'ReflectionException' with message 'Class App\Http\Controllers\Backend\AuthController does not exist' in D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Route.php:280
Stack trace:
#0 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Route.php(280): ReflectionMethod->__construct('App\\Http\\Contro...', 'showLoginForm')
#1 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Router.php(859): Illuminate\Routing\Route->signatureParameters('Illuminate\\Data...')
#2 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Router.php(844): Illuminate\Routing\Router->substituteImplicitBindings(Object(Illuminate\Routing\Route))
#3 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Router.php(827): Illuminate\Routing\Router->substituteBindings(Object(Illuminate\Routing\Route))
#4 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Router.php(691): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#5 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Router.php(675): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#6 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(246): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#7 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#8 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(52): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#9 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(44): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#11 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(136): call_user_func_array(Array, Array)
#12 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#13 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#15 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#16 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(132): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#17 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#18 D:\xampp\htdocs\raovatvnn\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#19 {main}  
[2016-06-16 09:49:57] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Encryption\EncryptionServiceProvider.php:45
Stack trace:
#0 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Encryption\EncryptionServiceProvider.php(25): Illuminate\Encryption\EncryptionServiceProvider->getEncrypterForKeyAndCipher(NULL, 'AES-256-CBC')
#1 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Container\Container.php(735): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#2 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Container\Container.php(633): Illuminate\Container\Container->build(Object(Closure), Array)
#3 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(697): Illuminate\Container\Container->make('encrypter', Array)
#4 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Container\Container.php(853): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#5 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Container\Container.php(808): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#6 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Container\Container.php(779): Illuminate\Container\Container->getDependencies(Array, Array)
#7 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Container\Container.php(633): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#8 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(697): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#9 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(126): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#10 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#11 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#12 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#13 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Router.php(726): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Router.php(699): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#16 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Router.php(675): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#17 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(246): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#18 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#19 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(52): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#20 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(44): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#21 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#22 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(136): call_user_func_array(Array, Array)
#23 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#24 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#25 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#26 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#27 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(132): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#28 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#29 D:\xampp\htdocs\raovatvnn\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#30 {main}  
[2016-06-16 09:49:57] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Encryption\EncryptionServiceProvider.php:45
Stack trace:
#0 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Encryption\EncryptionServiceProvider.php(25): Illuminate\Encryption\EncryptionServiceProvider->getEncrypterForKeyAndCipher(NULL, 'AES-256-CBC')
#1 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Container\Container.php(735): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#2 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Container\Container.php(633): Illuminate\Container\Container->build(Object(Closure), Array)
#3 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(697): Illuminate\Container\Container->make('encrypter', Array)
#4 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Container\Container.php(853): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#5 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Container\Container.php(808): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#6 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Container\Container.php(779): Illuminate\Container\Container->getDependencies(Array, Array)
#7 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Container\Container.php(633): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#8 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(697): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#9 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(152): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#10 D:\xampp\htdocs\raovatvnn\index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#11 {main}  
[2016-06-17 03:07:25] local.ERROR: exception 'ErrorException' with message 'Missing argument 1 for Illuminate\Database\Schema\Blueprint::timestamp(), called in D:\xampp\htdocs\raovatvnn\protected\database\migrations\2014_10_12_000000_create_users_table.php on line 22 and defined' in D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Database\Schema\Blueprint.php:759
Stack trace:
#0 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Database\Schema\Blueprint.php(759): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(2, 'Missing argumen...', 'D:\\xampp\\htdocs...', 759, Array)
#1 D:\xampp\htdocs\raovatvnn\protected\database\migrations\2014_10_12_000000_create_users_table.php(22): Illuminate\Database\Schema\Blueprint->timestamp()
#2 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Database\Schema\Builder.php(146): CreateUsersTable->{closure}(Object(Illuminate\Database\Schema\Blueprint))
#3 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(221): Illuminate\Database\Schema\Builder->create('users', Object(Closure))
#4 D:\xampp\htdocs\raovatvnn\protected\database\migrations\2014_10_12_000000_create_users_table.php(24): Illuminate\Support\Facades\Facade::__callStatic('create', Array)
#5 D:\xampp\htdocs\raovatvnn\protected\database\migrations\2014_10_12_000000_create_users_table.php(24): Illuminate\Support\Facades\Schema::create('users', Object(Closure))
#6 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Database\Migrations\Migrator.php(147): CreateUsersTable->up()
#7 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Database\Migrations\Migrator.php(117): Illuminate\Database\Migrations\Migrator->runUp('2014_10_12_0000...', 1, false)
#8 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Database\Migrations\Migrator.php(86): Illuminate\Database\Migrations\Migrator->runMigrationList(Array, Array)
#9 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Database\Console\Migrations\MigrateCommand.php(77): Illuminate\Database\Migrations\Migrator->run('D:\\xampp\\htdocs...', Array)
#10 [internal function]: Illuminate\Database\Console\Migrations\MigrateCommand->fire()
#11 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Container\Container.php(507): call_user_func_array(Array, Array)
#12 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Console\Command.php(169): Illuminate\Container\Container->call(Array)
#13 D:\xampp\htdocs\raovatvnn\protected\vendor\symfony\console\Command\Command.php(256): Illuminate\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#14 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Console\Command.php(155): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#15 D:\xampp\htdocs\raovatvnn\protected\vendor\symfony\console\Application.php(791): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#16 D:\xampp\htdocs\raovatvnn\protected\vendor\symfony\console\Application.php(186): Symfony\Component\Console\Application->doRunCommand(Object(Illuminate\Database\Console\Migrations\MigrateCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#17 D:\xampp\htdocs\raovatvnn\protected\vendor\symfony\console\Application.php(117): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#18 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(107): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#19 D:\xampp\htdocs\raovatvnn\protected\artisan(36): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#20 {main}  
[2016-06-17 03:08:57] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'CreateAdminsRoleTable' not found' in D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Database\Migrations\Migrator.php:336
Stack trace:
#0 {main}  
[2016-06-17 08:15:40] local.ERROR: exception 'BadMethodCallException' with message 'Method add does not exist.' in D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Support\Traits\Macroable.php:81
Stack trace:
#0 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(221): Illuminate\Routing\Router->__call('add', Array)
#1 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(221): Illuminate\Routing\Router->add('add', Array)
#2 D:\xampp\htdocs\raovatvnn\protected\app\Http\routes.php(38): Illuminate\Support\Facades\Facade::__callStatic('add', Array)
#3 D:\xampp\htdocs\raovatvnn\protected\app\Http\routes.php(38): Illuminate\Support\Facades\Route::add('add', Array)
#4 [internal function]: App\Providers\RouteServiceProvider->{closure}(Object(Illuminate\Routing\Router))
#5 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Router.php(398): call_user_func(Object(Closure), Object(Illuminate\Routing\Router))
#6 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(221): Illuminate\Routing\Router->group(Array, Object(Closure))
#7 D:\xampp\htdocs\raovatvnn\protected\app\Http\routes.php(39): Illuminate\Support\Facades\Facade::__callStatic('group', Array)
#8 D:\xampp\htdocs\raovatvnn\protected\app\Http\routes.php(39): Illuminate\Support\Facades\Route::group(Array, Object(Closure))
#9 [internal function]: App\Providers\RouteServiceProvider->{closure}(Object(Illuminate\Routing\Router))
#10 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Router.php(398): call_user_func(Object(Closure), Object(Illuminate\Routing\Router))
#11 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(221): Illuminate\Routing\Router->group(Array, Object(Closure))
#12 D:\xampp\htdocs\raovatvnn\protected\app\Http\routes.php(40): Illuminate\Support\Facades\Facade::__callStatic('group', Array)
#13 D:\xampp\htdocs\raovatvnn\protected\app\Http\routes.php(40): Illuminate\Support\Facades\Route::group(Array, Object(Closure))
#14 D:\xampp\htdocs\raovatvnn\protected\app\Providers\RouteServiceProvider.php(58): require('D:\\xampp\\htdocs...')
#15 [internal function]: App\Providers\RouteServiceProvider->App\Providers\{closure}(Object(Illuminate\Routing\Router))
#16 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Router.php(398): call_user_func(Object(Closure), Object(Illuminate\Routing\Router))
#17 D:\xampp\htdocs\raovatvnn\protected\app\Providers\RouteServiceProvider.php(59): Illuminate\Routing\Router->group(Array, Object(Closure))
#18 D:\xampp\htdocs\raovatvnn\protected\app\Providers\RouteServiceProvider.php(40): App\Providers\RouteServiceProvider->mapWebRoutes(Object(Illuminate\Routing\Router))
#19 [internal function]: App\Providers\RouteServiceProvider->map(Object(Illuminate\Routing\Router))
#20 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Container\Container.php(507): call_user_func_array(Array, Array)
#21 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Support\Providers\RouteServiceProvider.php(72): Illuminate\Container\Container->call(Array)
#22 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Support\Providers\RouteServiceProvider.php(31): Illuminate\Foundation\Support\Providers\RouteServiceProvider->loadRoutes()
#23 D:\xampp\htdocs\raovatvnn\protected\app\Providers\RouteServiceProvider.php(29): Illuminate\Foundation\Support\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#24 [internal function]: App\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#25 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Container\Container.php(507): call_user_func_array(Array, Array)
#26 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(757): Illuminate\Container\Container->call(Array)
#27 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(740): Illuminate\Foundation\Application->bootProvider(Object(App\Providers\RouteServiceProvider))
#28 [internal function]: Illuminate\Foundation\Application->Illuminate\Foundation\{closure}(Object(App\Providers\RouteServiceProvider), 14)
#29 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(741): array_walk(Array, Object(Closure))
#30 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\BootProviders.php(17): Illuminate\Foundation\Application->boot()
#31 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(203): Illuminate\Foundation\Bootstrap\BootProviders->bootstrap(Object(Illuminate\Foundation\Application))
#32 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(232): Illuminate\Foundation\Application->bootstrapWith(Array)
#33 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(127): Illuminate\Foundation\Http\Kernel->bootstrap()
#34 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#35 D:\xampp\htdocs\raovatvnn\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#36 {main}  
[2016-06-17 08:16:07] local.ERROR: exception 'BadMethodCallException' with message 'Method add does not exist.' in D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Support\Traits\Macroable.php:81
Stack trace:
#0 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(221): Illuminate\Routing\Router->__call('add', Array)
#1 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(221): Illuminate\Routing\Router->add('add', Array)
#2 D:\xampp\htdocs\raovatvnn\protected\app\Http\routes.php(38): Illuminate\Support\Facades\Facade::__callStatic('add', Array)
#3 D:\xampp\htdocs\raovatvnn\protected\app\Http\routes.php(38): Illuminate\Support\Facades\Route::add('add', Array)
#4 [internal function]: App\Providers\RouteServiceProvider->{closure}(Object(Illuminate\Routing\Router))
#5 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Router.php(398): call_user_func(Object(Closure), Object(Illuminate\Routing\Router))
#6 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(221): Illuminate\Routing\Router->group(Array, Object(Closure))
#7 D:\xampp\htdocs\raovatvnn\protected\app\Http\routes.php(39): Illuminate\Support\Facades\Facade::__callStatic('group', Array)
#8 D:\xampp\htdocs\raovatvnn\protected\app\Http\routes.php(39): Illuminate\Support\Facades\Route::group(Array, Object(Closure))
#9 [internal function]: App\Providers\RouteServiceProvider->{closure}(Object(Illuminate\Routing\Router))
#10 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Router.php(398): call_user_func(Object(Closure), Object(Illuminate\Routing\Router))
#11 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(221): Illuminate\Routing\Router->group(Array, Object(Closure))
#12 D:\xampp\htdocs\raovatvnn\protected\app\Http\routes.php(40): Illuminate\Support\Facades\Facade::__callStatic('group', Array)
#13 D:\xampp\htdocs\raovatvnn\protected\app\Http\routes.php(40): Illuminate\Support\Facades\Route::group(Array, Object(Closure))
#14 D:\xampp\htdocs\raovatvnn\protected\app\Providers\RouteServiceProvider.php(58): require('D:\\xampp\\htdocs...')
#15 [internal function]: App\Providers\RouteServiceProvider->App\Providers\{closure}(Object(Illuminate\Routing\Router))
#16 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Router.php(398): call_user_func(Object(Closure), Object(Illuminate\Routing\Router))
#17 D:\xampp\htdocs\raovatvnn\protected\app\Providers\RouteServiceProvider.php(59): Illuminate\Routing\Router->group(Array, Object(Closure))
#18 D:\xampp\htdocs\raovatvnn\protected\app\Providers\RouteServiceProvider.php(40): App\Providers\RouteServiceProvider->mapWebRoutes(Object(Illuminate\Routing\Router))
#19 [internal function]: App\Providers\RouteServiceProvider->map(Object(Illuminate\Routing\Router))
#20 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Container\Container.php(507): call_user_func_array(Array, Array)
#21 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Support\Providers\RouteServiceProvider.php(72): Illuminate\Container\Container->call(Array)
#22 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Support\Providers\RouteServiceProvider.php(31): Illuminate\Foundation\Support\Providers\RouteServiceProvider->loadRoutes()
#23 D:\xampp\htdocs\raovatvnn\protected\app\Providers\RouteServiceProvider.php(29): Illuminate\Foundation\Support\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#24 [internal function]: App\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#25 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Container\Container.php(507): call_user_func_array(Array, Array)
#26 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(757): Illuminate\Container\Container->call(Array)
#27 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(740): Illuminate\Foundation\Application->bootProvider(Object(App\Providers\RouteServiceProvider))
#28 [internal function]: Illuminate\Foundation\Application->Illuminate\Foundation\{closure}(Object(App\Providers\RouteServiceProvider), 14)
#29 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(741): array_walk(Array, Object(Closure))
#30 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\BootProviders.php(17): Illuminate\Foundation\Application->boot()
#31 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(203): Illuminate\Foundation\Bootstrap\BootProviders->bootstrap(Object(Illuminate\Foundation\Application))
#32 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(232): Illuminate\Foundation\Application->bootstrapWith(Array)
#33 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(127): Illuminate\Foundation\Http\Kernel->bootstrap()
#34 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#35 D:\xampp\htdocs\raovatvnn\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#36 {main}  
[2016-06-17 08:16:37] local.ERROR: exception 'BadMethodCallException' with message 'Method add does not exist.' in D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Support\Traits\Macroable.php:81
Stack trace:
#0 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(221): Illuminate\Routing\Router->__call('add', Array)
#1 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(221): Illuminate\Routing\Router->add('add', Array)
#2 D:\xampp\htdocs\raovatvnn\protected\app\Http\routes.php(38): Illuminate\Support\Facades\Facade::__callStatic('add', Array)
#3 D:\xampp\htdocs\raovatvnn\protected\app\Http\routes.php(38): Illuminate\Support\Facades\Route::add('add', Array)
#4 [internal function]: App\Providers\RouteServiceProvider->{closure}(Object(Illuminate\Routing\Router))
#5 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Router.php(398): call_user_func(Object(Closure), Object(Illuminate\Routing\Router))
#6 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(221): Illuminate\Routing\Router->group(Array, Object(Closure))
#7 D:\xampp\htdocs\raovatvnn\protected\app\Http\routes.php(39): Illuminate\Support\Facades\Facade::__callStatic('group', Array)
#8 D:\xampp\htdocs\raovatvnn\protected\app\Http\routes.php(39): Illuminate\Support\Facades\Route::group(Array, Object(Closure))
#9 [internal function]: App\Providers\RouteServiceProvider->{closure}(Object(Illuminate\Routing\Router))
#10 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Router.php(398): call_user_func(Object(Closure), Object(Illuminate\Routing\Router))
#11 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(221): Illuminate\Routing\Router->group(Array, Object(Closure))
#12 D:\xampp\htdocs\raovatvnn\protected\app\Http\routes.php(40): Illuminate\Support\Facades\Facade::__callStatic('group', Array)
#13 D:\xampp\htdocs\raovatvnn\protected\app\Http\routes.php(40): Illuminate\Support\Facades\Route::group(Array, Object(Closure))
#14 D:\xampp\htdocs\raovatvnn\protected\app\Providers\RouteServiceProvider.php(58): require('D:\\xampp\\htdocs...')
#15 [internal function]: App\Providers\RouteServiceProvider->App\Providers\{closure}(Object(Illuminate\Routing\Router))
#16 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Router.php(398): call_user_func(Object(Closure), Object(Illuminate\Routing\Router))
#17 D:\xampp\htdocs\raovatvnn\protected\app\Providers\RouteServiceProvider.php(59): Illuminate\Routing\Router->group(Array, Object(Closure))
#18 D:\xampp\htdocs\raovatvnn\protected\app\Providers\RouteServiceProvider.php(40): App\Providers\RouteServiceProvider->mapWebRoutes(Object(Illuminate\Routing\Router))
#19 [internal function]: App\Providers\RouteServiceProvider->map(Object(Illuminate\Routing\Router))
#20 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Container\Container.php(507): call_user_func_array(Array, Array)
#21 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Support\Providers\RouteServiceProvider.php(72): Illuminate\Container\Container->call(Array)
#22 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Support\Providers\RouteServiceProvider.php(31): Illuminate\Foundation\Support\Providers\RouteServiceProvider->loadRoutes()
#23 D:\xampp\htdocs\raovatvnn\protected\app\Providers\RouteServiceProvider.php(29): Illuminate\Foundation\Support\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#24 [internal function]: App\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#25 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Container\Container.php(507): call_user_func_array(Array, Array)
#26 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(757): Illuminate\Container\Container->call(Array)
#27 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(740): Illuminate\Foundation\Application->bootProvider(Object(App\Providers\RouteServiceProvider))
#28 [internal function]: Illuminate\Foundation\Application->Illuminate\Foundation\{closure}(Object(App\Providers\RouteServiceProvider), 14)
#29 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(741): array_walk(Array, Object(Closure))
#30 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\BootProviders.php(17): Illuminate\Foundation\Application->boot()
#31 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(203): Illuminate\Foundation\Bootstrap\BootProviders->bootstrap(Object(Illuminate\Foundation\Application))
#32 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(232): Illuminate\Foundation\Application->bootstrapWith(Array)
#33 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(127): Illuminate\Foundation\Http\Kernel->bootstrap()
#34 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#35 D:\xampp\htdocs\raovatvnn\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#36 {main}  
[2016-06-17 08:16:57] local.ERROR: exception 'BadMethodCallException' with message 'Method add does not exist.' in D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Support\Traits\Macroable.php:81
Stack trace:
#0 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(221): Illuminate\Routing\Router->__call('add', Array)
#1 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(221): Illuminate\Routing\Router->add('add', Array)
#2 D:\xampp\htdocs\raovatvnn\protected\app\Http\routes.php(38): Illuminate\Support\Facades\Facade::__callStatic('add', Array)
#3 D:\xampp\htdocs\raovatvnn\protected\app\Http\routes.php(38): Illuminate\Support\Facades\Route::add('add', Array)
#4 [internal function]: App\Providers\RouteServiceProvider->{closure}(Object(Illuminate\Routing\Router))
#5 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Router.php(398): call_user_func(Object(Closure), Object(Illuminate\Routing\Router))
#6 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(221): Illuminate\Routing\Router->group(Array, Object(Closure))
#7 D:\xampp\htdocs\raovatvnn\protected\app\Http\routes.php(39): Illuminate\Support\Facades\Facade::__callStatic('group', Array)
#8 D:\xampp\htdocs\raovatvnn\protected\app\Http\routes.php(39): Illuminate\Support\Facades\Route::group(Array, Object(Closure))
#9 [internal function]: App\Providers\RouteServiceProvider->{closure}(Object(Illuminate\Routing\Router))
#10 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Router.php(398): call_user_func(Object(Closure), Object(Illuminate\Routing\Router))
#11 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(221): Illuminate\Routing\Router->group(Array, Object(Closure))
#12 D:\xampp\htdocs\raovatvnn\protected\app\Http\routes.php(40): Illuminate\Support\Facades\Facade::__callStatic('group', Array)
#13 D:\xampp\htdocs\raovatvnn\protected\app\Http\routes.php(40): Illuminate\Support\Facades\Route::group(Array, Object(Closure))
#14 D:\xampp\htdocs\raovatvnn\protected\app\Providers\RouteServiceProvider.php(58): require('D:\\xampp\\htdocs...')
#15 [internal function]: App\Providers\RouteServiceProvider->App\Providers\{closure}(Object(Illuminate\Routing\Router))
#16 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Router.php(398): call_user_func(Object(Closure), Object(Illuminate\Routing\Router))
#17 D:\xampp\htdocs\raovatvnn\protected\app\Providers\RouteServiceProvider.php(59): Illuminate\Routing\Router->group(Array, Object(Closure))
#18 D:\xampp\htdocs\raovatvnn\protected\app\Providers\RouteServiceProvider.php(40): App\Providers\RouteServiceProvider->mapWebRoutes(Object(Illuminate\Routing\Router))
#19 [internal function]: App\Providers\RouteServiceProvider->map(Object(Illuminate\Routing\Router))
#20 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Container\Container.php(507): call_user_func_array(Array, Array)
#21 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Support\Providers\RouteServiceProvider.php(72): Illuminate\Container\Container->call(Array)
#22 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Support\Providers\RouteServiceProvider.php(31): Illuminate\Foundation\Support\Providers\RouteServiceProvider->loadRoutes()
#23 D:\xampp\htdocs\raovatvnn\protected\app\Providers\RouteServiceProvider.php(29): Illuminate\Foundation\Support\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#24 [internal function]: App\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#25 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Container\Container.php(507): call_user_func_array(Array, Array)
#26 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(757): Illuminate\Container\Container->call(Array)
#27 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(740): Illuminate\Foundation\Application->bootProvider(Object(App\Providers\RouteServiceProvider))
#28 [internal function]: Illuminate\Foundation\Application->Illuminate\Foundation\{closure}(Object(App\Providers\RouteServiceProvider), 14)
#29 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(741): array_walk(Array, Object(Closure))
#30 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\BootProviders.php(17): Illuminate\Foundation\Application->boot()
#31 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(203): Illuminate\Foundation\Bootstrap\BootProviders->bootstrap(Object(Illuminate\Foundation\Application))
#32 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(232): Illuminate\Foundation\Application->bootstrapWith(Array)
#33 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(127): Illuminate\Foundation\Http\Kernel->bootstrap()
#34 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#35 D:\xampp\htdocs\raovatvnn\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#36 {main}  
[2016-06-17 08:20:46] local.ERROR: exception 'BadMethodCallException' with message 'Method add does not exist.' in D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Support\Traits\Macroable.php:81
Stack trace:
#0 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(221): Illuminate\Routing\Router->__call('add', Array)
#1 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(221): Illuminate\Routing\Router->add('add', Array)
#2 D:\xampp\htdocs\raovatvnn\protected\app\Http\routes.php(38): Illuminate\Support\Facades\Facade::__callStatic('add', Array)
#3 D:\xampp\htdocs\raovatvnn\protected\app\Http\routes.php(38): Illuminate\Support\Facades\Route::add('add', Array)
#4 [internal function]: App\Providers\RouteServiceProvider->{closure}(Object(Illuminate\Routing\Router))
#5 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Router.php(398): call_user_func(Object(Closure), Object(Illuminate\Routing\Router))
#6 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(221): Illuminate\Routing\Router->group(Array, Object(Closure))
#7 D:\xampp\htdocs\raovatvnn\protected\app\Http\routes.php(39): Illuminate\Support\Facades\Facade::__callStatic('group', Array)
#8 D:\xampp\htdocs\raovatvnn\protected\app\Http\routes.php(39): Illuminate\Support\Facades\Route::group(Array, Object(Closure))
#9 [internal function]: App\Providers\RouteServiceProvider->{closure}(Object(Illuminate\Routing\Router))
#10 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Router.php(398): call_user_func(Object(Closure), Object(Illuminate\Routing\Router))
#11 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(221): Illuminate\Routing\Router->group(Array, Object(Closure))
#12 D:\xampp\htdocs\raovatvnn\protected\app\Http\routes.php(40): Illuminate\Support\Facades\Facade::__callStatic('group', Array)
#13 D:\xampp\htdocs\raovatvnn\protected\app\Http\routes.php(40): Illuminate\Support\Facades\Route::group(Array, Object(Closure))
#14 D:\xampp\htdocs\raovatvnn\protected\app\Providers\RouteServiceProvider.php(58): require('D:\\xampp\\htdocs...')
#15 [internal function]: App\Providers\RouteServiceProvider->App\Providers\{closure}(Object(Illuminate\Routing\Router))
#16 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Router.php(398): call_user_func(Object(Closure), Object(Illuminate\Routing\Router))
#17 D:\xampp\htdocs\raovatvnn\protected\app\Providers\RouteServiceProvider.php(59): Illuminate\Routing\Router->group(Array, Object(Closure))
#18 D:\xampp\htdocs\raovatvnn\protected\app\Providers\RouteServiceProvider.php(40): App\Providers\RouteServiceProvider->mapWebRoutes(Object(Illuminate\Routing\Router))
#19 [internal function]: App\Providers\RouteServiceProvider->map(Object(Illuminate\Routing\Router))
#20 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Container\Container.php(507): call_user_func_array(Array, Array)
#21 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Support\Providers\RouteServiceProvider.php(72): Illuminate\Container\Container->call(Array)
#22 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Support\Providers\RouteServiceProvider.php(31): Illuminate\Foundation\Support\Providers\RouteServiceProvider->loadRoutes()
#23 D:\xampp\htdocs\raovatvnn\protected\app\Providers\RouteServiceProvider.php(29): Illuminate\Foundation\Support\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#24 [internal function]: App\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#25 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Container\Container.php(507): call_user_func_array(Array, Array)
#26 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(757): Illuminate\Container\Container->call(Array)
#27 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(740): Illuminate\Foundation\Application->bootProvider(Object(App\Providers\RouteServiceProvider))
#28 [internal function]: Illuminate\Foundation\Application->Illuminate\Foundation\{closure}(Object(App\Providers\RouteServiceProvider), 14)
#29 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(741): array_walk(Array, Object(Closure))
#30 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\BootProviders.php(17): Illuminate\Foundation\Application->boot()
#31 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(203): Illuminate\Foundation\Bootstrap\BootProviders->bootstrap(Object(Illuminate\Foundation\Application))
#32 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(232): Illuminate\Foundation\Application->bootstrapWith(Array)
#33 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(127): Illuminate\Foundation\Http\Kernel->bootstrap()
#34 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#35 D:\xampp\htdocs\raovatvnn\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#36 {main}  
[2016-06-17 08:20:48] local.ERROR: exception 'BadMethodCallException' with message 'Method add does not exist.' in D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Support\Traits\Macroable.php:81
Stack trace:
#0 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(221): Illuminate\Routing\Router->__call('add', Array)
#1 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(221): Illuminate\Routing\Router->add('add', Array)
#2 D:\xampp\htdocs\raovatvnn\protected\app\Http\routes.php(38): Illuminate\Support\Facades\Facade::__callStatic('add', Array)
#3 D:\xampp\htdocs\raovatvnn\protected\app\Http\routes.php(38): Illuminate\Support\Facades\Route::add('add', Array)
#4 [internal function]: App\Providers\RouteServiceProvider->{closure}(Object(Illuminate\Routing\Router))
#5 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Router.php(398): call_user_func(Object(Closure), Object(Illuminate\Routing\Router))
#6 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(221): Illuminate\Routing\Router->group(Array, Object(Closure))
#7 D:\xampp\htdocs\raovatvnn\protected\app\Http\routes.php(39): Illuminate\Support\Facades\Facade::__callStatic('group', Array)
#8 D:\xampp\htdocs\raovatvnn\protected\app\Http\routes.php(39): Illuminate\Support\Facades\Route::group(Array, Object(Closure))
#9 [internal function]: App\Providers\RouteServiceProvider->{closure}(Object(Illuminate\Routing\Router))
#10 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Router.php(398): call_user_func(Object(Closure), Object(Illuminate\Routing\Router))
#11 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(221): Illuminate\Routing\Router->group(Array, Object(Closure))
#12 D:\xampp\htdocs\raovatvnn\protected\app\Http\routes.php(40): Illuminate\Support\Facades\Facade::__callStatic('group', Array)
#13 D:\xampp\htdocs\raovatvnn\protected\app\Http\routes.php(40): Illuminate\Support\Facades\Route::group(Array, Object(Closure))
#14 D:\xampp\htdocs\raovatvnn\protected\app\Providers\RouteServiceProvider.php(58): require('D:\\xampp\\htdocs...')
#15 [internal function]: App\Providers\RouteServiceProvider->App\Providers\{closure}(Object(Illuminate\Routing\Router))
#16 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Routing\Router.php(398): call_user_func(Object(Closure), Object(Illuminate\Routing\Router))
#17 D:\xampp\htdocs\raovatvnn\protected\app\Providers\RouteServiceProvider.php(59): Illuminate\Routing\Router->group(Array, Object(Closure))
#18 D:\xampp\htdocs\raovatvnn\protected\app\Providers\RouteServiceProvider.php(40): App\Providers\RouteServiceProvider->mapWebRoutes(Object(Illuminate\Routing\Router))
#19 [internal function]: App\Providers\RouteServiceProvider->map(Object(Illuminate\Routing\Router))
#20 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Container\Container.php(507): call_user_func_array(Array, Array)
#21 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Support\Providers\RouteServiceProvider.php(72): Illuminate\Container\Container->call(Array)
#22 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Support\Providers\RouteServiceProvider.php(31): Illuminate\Foundation\Support\Providers\RouteServiceProvider->loadRoutes()
#23 D:\xampp\htdocs\raovatvnn\protected\app\Providers\RouteServiceProvider.php(29): Illuminate\Foundation\Support\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#24 [internal function]: App\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#25 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Container\Container.php(507): call_user_func_array(Array, Array)
#26 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(757): Illuminate\Container\Container->call(Array)
#27 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(740): Illuminate\Foundation\Application->bootProvider(Object(App\Providers\RouteServiceProvider))
#28 [internal function]: Illuminate\Foundation\Application->Illuminate\Foundation\{closure}(Object(App\Providers\RouteServiceProvider), 14)
#29 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(741): array_walk(Array, Object(Closure))
#30 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\BootProviders.php(17): Illuminate\Foundation\Application->boot()
#31 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(203): Illuminate\Foundation\Bootstrap\BootProviders->bootstrap(Object(Illuminate\Foundation\Application))
#32 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(232): Illuminate\Foundation\Application->bootstrapWith(Array)
#33 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(127): Illuminate\Foundation\Http\Kernel->bootstrap()
#34 D:\xampp\htdocs\raovatvnn\protected\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#35 D:\xampp\htdocs\raovatvnn\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#36 {main}  
[2016-06-17 09:54:35] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected ';'' in D:\xampp\htdocs\raovatvnn\protected\storage\framework\views\6a61446d66bd7f56d634eae3cd7011f756e7e631.php:35
Stack trace:
#0 {main}  
